modules/standard/Path.chpl

```diff
@@ -495,7 +590,15 @@ private proc joinPath(paths: [] string): string {
 }
 
 // Normalize leading slash count to a value between 0 and 2.
-private proc normalizeLeadingSlashCount(name: string): int {
+private inline proc normalizeLeadingSlashCount(name: string): int {
+  return normalizeLeadingSlashCountImpl(name);
+}
+
+private inline proc normalizeLeadingSlashCount(name: bytes): int {
+  return normalizeLeadingSlashCountImpl(name);
+}
+
+private proc normalizeLeadingSlashCountImpl(name: ?t): int {
```

Comment:
In the below, I recommend you create `private proc getPathSep(type t)` and do `const pathSep = getPathSep(t)`, where that returns "/" as a bytes or string according to t.